setup.py

```diff
@@ -14,6 +14,16 @@
 
 _atari_deps = ["gymnasium[atari, accept-rom-license]"]
 _mujoco_deps = ["gymnasium[mujoco]", "mujoco<2.5"]
+_nethack_deps = [
+    "numba ~= 0.58",
+    "pandas ~= 2.1",
+    "matplotlib ~= 3.8",
+    "seaborn ~= 0.12",
+    "scipy ~= 1.11",
+    "shimmy",
+    "tqdm ~= 4.66",
+    "debugpy ~= 1.6",
```

Comment:
Today I learned!
Didn't know about this `~=` feature

Reply:
This is actually a useful feature. It allows bug fix releases but avoids introducing new features that could potentially break your code.